docs/tutorial/mlflow-kubeflow.rst

```diff
@@ -100,38 +96,40 @@ This will periodically run a ``juju status`` command and filter to components wh
 
 Don't be surprised if some of the components' statuses change to ``blocked`` or ``error`` every now and then. This is expected behaviour, and these statuses should resolve by themselves as the bundle configures itself. However, if components remain stuck in the same error states, consult the troubleshooting steps below.
 
-.. dropdown:: Expand to troubleshoot: Waiting for gateway relation
+Be patient, it can take up to half an hour for all those charms to download and initialise. In the meantime, why not try our `Juju tutorial <https://juju.is/docs/juju/get-started-with-juju>`_?
 
-   An issue you might have is the ``tensorboard-controller`` component might be stuck with a status of ``waiting`` and a message “Waiting for gateway relation”. To fix this, run:
+Integrate MLflow with Kubeflow Dashboard
+----------------------------------------
+You can integrate your charmed MLflow deployment with Kubeflow dashboard by running following commands: 
 
-   .. code-block:: bash
+.. code-block:: bash
+
+   juju integrate mlflow-server:ingress istio-pilot:ingress
+   juju integrate mlflow-server:dashboard-links kubeflow-dashboard:links
 
-      juju run --unit istio-pilot/0 -- "export JUJU_DISPATCH_PATH=hooks/config-changed; ./dispatch"
+Now you should see the MLflow tab in the left sidebar of your Kubeflow Dashboard at ``http://10.64.140.43.nip.io/``.
```

Comment:
```suggestion
Now you should see the MLflow tab in the left sidebar of your Kubeflow dashboard at ``http://10.64.140.43.nip.io/``.
```
We use lowercase before for "dashboard". Is it an own name? Should we use CAPS instead?

Reply:
Can we add ``http://10.64.140.43.nip.io/`` as a code block so the user can copy and paste?